syncano/models/manager.py

```diff
@@ -828,6 +828,26 @@ def _get_endpoint_properties(self):
         return self.model._meta.resolve_endpoint(self.endpoint, defaults), defaults
 
 
+class RuntimeChoices(object):
```

Comment:
IMHO it's not needed. You just removed the validator from runtime_name, so why bother with this ?
Also why not simply return a list ? When you set attributes for an object you have to do dir(object) instead just print a list.